controllers/suite_test.go

```diff
@@ -50,6 +51,9 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
```

Comment:
why do we need all this?

Reply:
Setting the environment variables allows us running (and debugging)  the tests through the IDE and only through `make tests`